backend/src/main/java/se/chalmers/dat265/group1/WebApi.java

```diff
@@ -0,0 +1,142 @@
+package se.chalmers.dat265.group1;
+
+import se.chalmers.dat265.group1.api.MaterialInterface;
+import se.chalmers.dat265.group1.api.printing.PrintingController;
+import se.chalmers.dat265.group1.api.printing.PrintingInterface;
+import se.chalmers.dat265.group1.api.customer.CustomerAPI;
+import se.chalmers.dat265.group1.api.digitalpart.DigitalPartAPI;
+import se.chalmers.dat265.group1.api.digitalpart.DigitalPartController;
+import se.chalmers.dat265.group1.api.customer.CustomerController;
+import com.google.gson.Gson;
+import se.chalmers.dat265.group1.model.*;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import se.chalmers.dat265.group1.storage.repository.GenericRepository;
+
+import static spark.Spark.*;
+
+/**
+ * Starts a restapi att localhost:4567
+ */
+public class WebApi {
+
+
+    private static CustomerAPI ci;
+    private static DigitalPartAPI dpi;
+    private static MaterialInterface mi;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "mi" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)